app/views/projects/members/index.html.erb

```diff
@@ -1,36 +1,104 @@
-<%= render Viral::PageHeaderComponent.new(title: t('.title'), subtitle: t('.subtitle', namespace_type: @namespace.class.model_name.human, namespace_name: @namespace.name)) do |component| %>
+<%= render Viral::PageHeaderComponent.new(title: t(:'.title'), subtitle: t(:'.subtitle', namespace_type: @namespace.class.model_name.human, namespace_name: @namespace.name)) do |component| %>
   <%= component.with_icon(name: "users", classes: "h-14 w-14 text-primary-700") %>
   <%= component.with_buttons do %>
     <% if allowed_to?(:create_member?, @namespace) %>
       <%= link_to t(:".add"),
-      new_namespace_project_member_path,
-      data: {
-        turbo_frame: "new_member_modal",
-        turbo_stream: true
-      },
-      class: "button button--state-primary button--size-default",
-      "aria-label": t(:".actions.button_add_aria_label") %>
+                  new_namespace_project_member_path(@namespace.parent, @namespace.project, **request.query_parameters),
```

Comment:
**nitpick (blocking):** Same comment as above

Reply:
Updated in  9d14e974555ff0bbe5c4b392f2e0b47b8a2b1865